components/ide/jetbrains/backend-plugin/src/main/resources-latest/META-INF/extensions.xml

```diff
@@ -7,6 +7,18 @@
 <idea-plugin>
     <extensions defaultExtensionNs="com.intellij">
         <applicationService serviceInterface="io.gitpod.jetbrains.remote.GitpodIgnoredPortsForNotificationService" serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodIgnoredPortsForNotificationServiceImpl" preload="true"/>
-        <projectService serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodPortForwardingService" preload="true" client="guest"/>
+        <applicationService serviceInterface="io.gitpod.jetbrains.remote.GitpodCLIHelper" serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodCLIHelperImpl"/>
+        <applicationService serviceInterface="io.gitpod.jetbrains.remote.GitpodGlobalPortForwardingService" serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodGlobalPortForwardingServiceImpl"/>
+        <projectService serviceImplementation="io.gitpod.jetbrains.remote.GitpodClientProjectSessionTracker" client="controller" preload="true"/>
+        <projectService serviceImplementation="io.gitpod.jetbrains.remote.GitpodTerminalService" client="controller" preload="true"/>
+        <projectService serviceImplementation="io.gitpod.jetbrains.remote.latest.GitpodPortForwardingService" client="controller" preload="true"/>
     </extensions>
+    <actions>
+        <action id="io.gitpod.jetbrains.remote.latest.GitpodPortsActionCopyUrl"
+                class="io.gitpod.jetbrains.remote.latest.GitpodPortsActionCopyUrl"
+                text="Gitpod: Copy Port URL"
+                icon="AllIcons.Actions.Copy">
+            <add-to-group group-id="PortForwardingPortGroup" anchor="last"/>
```

Comment:
💡 Nice finding, Andrea!

ℹ️  I've taken screenshots as a reminder of the difference between the two PortForwarding group ids:
- `PortForwardingPortGroup`:
  <img width="574" alt="image" src="https://user-images.githubusercontent.com/418083/197352726-375d8995-9a7c-48f5-97e6-a5cfe9e1fff5.png">

- `PortForwardingSuggestionGroup`:
  <img width="457" alt="image" src="https://user-images.githubusercontent.com/418083/197352729-3183c1d2-b893-45af-8d3a-2887e1a624ce.png">

💭  This also indicates that this grey line in the terminal is related to "Port Forwarding Suggestion", so maybe we can find another class that manages this suggestion's display.